pkgs/by-name/dc/dc3dd/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchzip
+, perlPackages
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dc3dd";
+  version = "7.3.1";
+
+  src = fetchzip {
+    url = "mirror://sourceforge/dc3dd/dc3dd-${finalAttrs.version}.zip";
+    hash = "sha256-SYDoqGlsROHX1a0jJX11F+yp6CeFK+tZbYOOnScC6Ig=";
+  };
+
```

Comment:
Extra hint:
```suggestion

  outputs = [ "out" "man" ];

```

Reply:
Is it used for what?

Reply:
For creating two "packages", the one containing the files themselves and the other containing the manpages only.

This is useful, among many things, to reduce the size of installed filesets (minimal ISOs were a typical use case) and for some special applications like online manpage renderers.